components/text-utilities/text-utilities.css

```diff
@@ -0,0 +1,23 @@
+.txt-small{
+    font-size: small;
+}
+
+.txt-gray{
+    color: var(--darkgrey);
+   font-weight: bold;
+}
+.txt-align{
```

Comment:
can you elaborate what text align means here ? i mean if you are using glex for alignment you can keep them in flex helper class IMO.

Reply:
Actually I needed a div inorder to show different alignment of text. So just added a class for it.